### ISSUE TITLE
StepDown function not working

### ISSUE THREAD

**Issue by thomascoppein** [hyperform/hyperform#118]
The stepDown function on a number input is not working. The stepUp function works perfect. 
The is no error or warning shown.

**Comment by Boldewyn** [comment]
Oh. Thanks for the report! I’ll investigate.

**Comment by Boldewyn** [comment]
It’s this line, but I need to figure out again, how exactly it supports dates, too, before I can change it:

https://github.com/hyperform/hyperform/blob/master/src/tools/get_next_valid.js#L70

**Comment by Boldewyn** [comment]
Addendum: Apparently one should not try to use the number feature of `stepDown()` at the moment unless they are interested in ridiculously wrong results.

**Comment by dietervyncke** [comment]
Is there any update on this topic. Can these fixes be released?

**Comment by Boldewyn** [comment]
Yes, thanks for bringing this to my attention again! In principle the fix is ready but I wasn’t able to test it appropriately (and generalize it as much as I wanted). The problem is, that stepUp/stepDown isn’t yet working reliably with `type=date` and related inputs yet, and I wanted to fix those as well with the same release. If I can’t make that working this week, I’ll do a “partial release” of the current fix.
